.coafile

```diff
@@ -0,0 +1,26 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, tmp/**, .git/**
+
+[all.SpaceConsistencyBear]
+files = **.js, **.json, **.yml, **.md, .eslintrc.js
```

Comment:
You do not need to add .eslintrc.js

I removed it, and it was included because of `**.js`

see https://travis-ci.org/jayvdb/gsoc-prep-tasks/jobs/391075675#L741